shared/oval/sshd_use_approved_ciphers.xml

```diff
@@ -4,12 +4,12 @@
       <title>Use Only Approved Ciphers</title>
       <affected family="unix">
         <platform>multi_platform_rhel</platform>
+        <platform>multi_platform_debian</platform>
       </affected>
       <description>Limit the ciphers to those which are FIPS-approved and only
       use ciphers in counter (CTR) mode.</description>
     </metadata>
     <criteria operator="AND">
-      <extend_definition comment="Installed OS is certified" definition_ref="installed_OS_is_certified" />
```

Comment:
I'll add new xccdf/oval definitions (`sshd_use_strong_ciphers` and `sshd_use_strong_macs`) without FIPS then.